client/my-sites/people/invite-people/index.jsx

```diff
@@ -114,6 +116,7 @@ export default React.createClass( {
 							siteId={ this.props.site.ID }
 							valueLink={ this.linkState( 'role' ) }
 							disabled={ this.state.sendingInvites }
+							appendRoles={ { follower: {} } } // adds a pseudo-role of follower
```

Comment:
It feels a bit awkward to pass in an object with just a single key. What do you think of either:

- Including an `addFollower` prop to RoleSelect and use that
- Pass in an array of roles, and then in `refreshRoles()` do something like:

```
this.setState( {
	roles: Object.keys( siteRoles ).concat( appendRoles )
} );
```

That snippet is untested, so may not work right :stuck_out_tongue_closed_eyes: